scripts/camkes.sh

```diff
@@ -43,7 +44,6 @@ as_root apt-get install -y --no-install-recommends \
     linux-libc-dev:i386 \
     pkg-config \
     spin \
-    lib32stdc++-10-dev \
```

Comment:
I am not sure if this is needed for x86 container either?

Reply:
yes, should be good to remove